packages/xrpl/src/models/transactions/checkCash.ts

```diff
@@ -53,12 +53,14 @@ export function validateCheckCash(tx: Record<string, unknown>): void {
     )
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- Necessary check
```

Comment:
We can probably disable this check in `src/models/transactions`, since it's the whole point of `validate`

Reply:
@mvadari Can you elaborate?

Reply:
In the eslint settings, we should disable this change for this folder. Doesn't need to happen in this PR, just an idle thought.